README.md

```
@@ -383,6 +384,43 @@ logger.addSink<minta::RollingFileSink>(
+try {
+    riskyDatabaseQuery();
+} catch (const std::exception& ex) {
+    // Exception only — uses ex.what() as the message
+    logger.error(ex);
+}
+```
+
+All six levels have exception overloads: `trace`, `debug`, `info`, `warn`, `error`, `fatal`. Nested exceptions are unwound via `std::rethrow_if_nested` (up to 20 levels deep), and exception type names are demangled on GCC/Clang via `abi::__cxa_demangle`.
+
+**Human-readable output:**
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)